src/plugins/drawing-tool/components/drawing-tool.tsx

```diff
@@ -47,6 +49,10 @@ const DrawingToolComponent: React.FC<IProps> = (props) => {
     });
   };
 
+  const handleDelete = () => {
+    contentRef.current.deleteObjects(contentRef.current.selectedIds);
```

Comment:
the old handle delete checked for readOnly mode. Do we not need that here because now that it is a button, the click handler won't fire if it is disabled?

Reply:
Thanks for finding this. I tested it and it is possible to both paste images and delete things in readOnly mode. It takes some trickery to get the object selected so you can delete it. So I added a conditional to fix that.